hw/ip/i2c/data/i2c.hjson

```diff
@@ -645,7 +645,14 @@
         }
         { bits: "10:8"
           name: "SIGNAL"
-          desc: "Host issued a START before transmitting ABYTE, a STOP or a RESTART after the preceeding ABYTE"
+          desc: '''
+                Host issued a START before transmitting ABYTE, a STOP or a RESTART after the preceeding ABYTE
```

Comment:
nit: This first sentence doesn't make much sense.

Reply:
Fixed up all the descriptions.